packages/headless/src/api/search/search/stemming.ts

```diff
@@ -4,3 +4,12 @@
  * E.g., for the query "indexing consultant", where a result contains "Techzample is a consulting firm specializing in search indexes. It employs consultants that can analyze and improve your search index.", the `TermsToHighlight` might be `{ "indexing": ["indexes", "index"], "consultant": ["consulting", "consultants"] }`.
  */
 export type TermsToHighlight = {[originalTerm: string]: string[]};
+
+/**
+ * Phrases to highlight (with [stemming](https://docs.coveo.com/en/1576)) in the results.
+ *
+ * E.g., for the exact phrase "indexing consultant", where a result contains "Techzample is a consulting firm specializing in search indexes. It employs consultants that can analyze and improve your search index.", the `PhrasesToHighlight` might be `{ "indexing consultant": {"indexing": ["indexes", "index"], "consultant": ["consulting", "consultants"]}}`.
```

Comment:
A note on how it differs from Terms to highlight would be helpful. E.g. I am guessing its `{ "indexing consultant"` vs  ``{ "indexing"`? In that case isn't `"consultant":` a term?

Reply:
Got it, thanks